bindings/python/src/torrent_handle.cpp

```diff
@@ -191,10 +191,6 @@ void dict_to_announce_entry(dict d, announce_entry& ae)
       ae.tier = extract<int>(d["tier"]);
    if (d.has_key("fail_limit"))
       ae.fail_limit = extract<int>(d["fail_limit"]);
-   if (d.has_key("source"))
-      ae.source = extract<int>(d["source"]);
```

Comment:
if this is in fact used for adding new trackers, the "source" fields needs to be forwarded